cmd/preflight/cli/interactive_results.go

```diff
@@ -217,8 +217,20 @@ func estimateNumberOfLines(text string, width int) int {
 	return lines
 }
 
-func save(preflightName string, analyzeResults []*analyzerunner.AnalyzeResult) (string, error) {
-	filename := path.Join(util.HomeDir(), fmt.Sprintf("%s-results.txt", preflightName))
+func save(preflightName string, outputPath string, analyzeResults []*analyzerunner.AnalyzeResult) (string, error) {
+	filename := ""
+	if outputPath != "" {
+		// use override output path
+		overridePath, err := convert.ValidateOutputPath(outputPath)
+		if err != nil {
+			return "", errors.Wrap(err, "override output file path")
+		}
+		filename = overridePath
+	} else {
+		// use default output path
+		filename = fmt.Sprintf("%s-results.txt", preflightName)
```

Comment:
Personally, I think it would be cool to use a timestamp in the filename to prevent overwrites, but this is just a suggestion.